consensus/babe/babe.go

```diff
@@ -107,7 +107,7 @@ func (b *Session) invokeBlockAuthoring() {
 			}
 
 			// Broadcast the block
-			blockAnnounceMsg := p2p.BlockAnnounceMessage{
+			blockAnnounceMsg := &p2p.BlockAnnounceMessage{
```

Comment:
I believe this needs to be the full header (BlockAnnounceMessage == Block Header)